### DISCUSSION TITLE
Multi language UI support

### DISCUSSION THREAD

**Discussion by AlexHladin** [navidrome/navidrome#2725] [Ideas]
The Navidrome has a lot of users worldwide, so I guess it is a good time to support multilanguage UI.

**Comment by deluan** [comment]
Something like this? :)

https://github.com/navidrome/navidrome/assets/331353/ea877eec-8ebd-44b3-ba82-c67470527eb4

**Reply by deluan** [reply]
By the way:

![Screenshot 2023-12-19 at 10 11 20 AM](https://github.com/navidrome/navidrome/assets/331353/24ad07a1-b4b5-436f-bdb7-702105e3667f)

**Comment by AlexHladin** [comment]
Awesome, I didn't know about this configuration. Thanks @deluan!
